src/microservices/Microservices.IsIdentifiable/Service/IsIdentifiableQueueConsumer.cs

```diff
@@ -30,13 +32,12 @@ protected override void ProcessMessageImpl(IMessageHeader header, BasicDeliverEv
             if (!SafeDeserializeToMessage(header, basicDeliverEventArgs, out ExtractFileStatusMessage message))
                 return;
 
-            // The path is taken from the message, however maybe it should be FileSystemOptions|ExtractRoot in default.yaml
             // If the filename has a rooted path then the ExtractionDirectory is ignored by Path.Combine
-            var toProcess = new FileInfo( Path.Combine(message.ExtractionDirectory, message.AnonymisedFileName) );
+            var toProcess = new FileInfo( Path.Combine(_extractionRoot, message.ExtractionDirectory, message.AnonymisedFileName) );
```

Comment:
Does this assume that all messages have the same extraction root? If we support hosting other imaging datasets e.g. retinopathy / pathology or project specific image sets under different root drives will this become a problem?  (i.e. should we be going the other way with this and taking it out of the yaml)?

Or is this the output directory...

Reply:
Yes, this is the output directory. The extraction directory still comes through in the message but this change now prefixes it with the extraction root directory so that it actually works. In future you're right it might need a different mechanism (could simply set the extraction root in the yaml to be '.' and have a full path in the message?). Although the extraction root directory is possibly conceptually on the user side of the fence rather than the system side, meaning it's more related to a 'project' than to a 'modality' in which case it won't need to change whatever modality/dataset is processed?

Reply:
Makes sense.  Happy to go with this for now then if it fixes something that was broken! Lets worry about the config setting vs message later then